### ISSUE TITLE
Descrepancy between URL and page title

### ISSUE THREAD

**Issue by norrishung** [chimecms/chime#536]
Chantrice was having trouble finding a page on her site in the CMS. I actually had the same problem until I realized it was because she renamed all of her topics and subtopics so the URL had no correlation with the actual IA of the site. 

This is something I can forsee being continually confusing.

**Comment by tmaybe** [comment]
This is something we've talked about but not crystalized in issue form before.

If a topic or article's URL changes every time its title is changed, it will result in broken links. We've talked about having some kind of automated redirect system in place, so that old URLs are redirected to the new ones, but that would get complicated. We could also invisibly change the URL if you change the title in the same activity you created the topic/article in, and warn otherwise?

I'm not sure what best practice for this sort of thing is in this case.

**Comment by migurski** [comment]
I would let the broken links stand.

**Comment by tmaybe** [comment]
Meaning, change the URL every time the title is changed and not worry about breaking links?

**Comment by migurski** [comment]
Yeah: have the URLs be consistent with what the user decides for that page.

**Comment by wpietri** [comment]
I agree, with the caveat that at some point on publish we'll need to start tracking URL changes so that we can redirect. And by "at some point" I mean "before we serve a real city web site". If we do our jobs right, cities will accumulate a ton of inbound links to useful information. I think we should try hard not to break those links.

**Comment by norrishung** [comment]
Update from Chantrice...

"Thanks, Norris! I got ya. Just FYI…That’s not actually what was confusing. I couldn’t remember which activity I started when I created that page. I clicked the “Getting Police Reports” activity and was confused when the old article “Requirements to become an NOPD officer” was still showing up instead of the updated article “Benefits & Requirements of becoming an NOPD officer.” I didn’t realize new articles wouldn’t automatically update within each activity. Thanks! Chantrice"
